Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAttackSurfaceReductionRulesPolicyWindows10ConfigManager/MSFT_IntuneAttackSurfaceReductionRulesPolicyWindows10ConfigManager.psm1

```diff
@@ -8,7 +8,7 @@ function Get-TargetResource
         [System.String]
         $Identity,
 
-        [Parameter()]
+        [Parameter(Mandatory = $True)]
```

Comment:
The parameter is already marked as Required in the *.schema.mof

Reply:
This resource was completely broken and not returning any of the settings.

@andikrueger Do you mean that I don't need to add the mandatory condition?

Reply:
Actually we need to declare the parameter as mandatory - basically this would match the schema.mof file. 

please review my second comment. There is something wrong with which parameter is the key for this resource and which is just mandatory in some cases.